internal/ethereum/event_enricher.go

```diff
@@ -96,9 +96,9 @@ func (ee *eventEnricher) filterEnrichEthLog(ctx context.Context, f *eventFilter,
 			Signature:        signature,
 			BlockHash:        ethLog.BlockHash.String(),
 			TransactionHash:  ethLog.TransactionHash.String(),
-			BlockNumber:      fftypes.FFuint64(blockNumber),
-			TransactionIndex: fftypes.FFuint64(transactionIndex),
-			LogIndex:         fftypes.FFuint64(logIndex),
```

Comment:
This is breaking change - with deprecated fields we keep it around for one release at least so we support both. Especially with a minor release.

Reply:
this line here is for a linter fix.

Reply:
the existing fields are still there but no longer used. 

The deprecation notes can be added during a release, it's also in the descriptions of those old fields.

Reply:
I see that, but they should still be used as a backup for a least a release so we do not break customers in a minor version. And then remove them in a later release such as https://github.com/hyperledger/firefly/pull/1289